docs/content/docs/dev/table/sqlClient.md

```diff
@@ -169,559 +177,523 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
 [INFO] Table update statement has been successfully submitted to the cluster:
 Cluster ID: StandaloneClusterId
 Job ID: 6f922fe5cba87406ff23ae4a7bb79044
-Web interface: http://localhost:8081
 ```
 
 <span class="label label-danger">Attention</span> The SQL Client does not track the status of the running Flink job after submission. The CLI process can be shutdown after the submission without affecting the detached query. Flink's `restart strategy` takes care of the fault-tolerance. A query can be cancelled using Flink's web interface, command-line, or REST API.
 
-{{< top >}}
+However, for batch users, it's more common that the next DML statement requires to wait util the
+previous DML statement finishes. In order to execute DML statements synchronously, you can set
+`table.dml-sync` option true in SQL Client.
 
-SQL Views
----------
+```sql
+Flink SQL> SET table.dml-sync = true;
+[INFO] Session property has been set.
+
+Flink SQL> INSERT INTO MyTableSink SELECT * FROM MyTableSource;
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] Execute statement in sync mode. Please wait for the execution finish...
+[INFO] Complete execution of the SQL update statement.
+```
 
-Views allow to define virtual tables from SQL queries. The view definition is parsed and validated immediately. However, the actual execution happens when the view is accessed during the submission of a general `INSERT INTO` or `SELECT` statement.
+<span class="label label-danger">Attention</span>  If you want to terminate the job, just type `CTRL-C` to cancel the execution.
 
-Views can either be defined in [environment files](sqlClient.html#environment-files) or within the CLI session.
+### Start a SQL Job from a savepoint
 
-The following example shows how to define multiple views in a file. The views are registered in the order in which they are defined in the environment file. Reference chains such as _view A depends on view B depends on view C_ are supported.
+Flink supports to start the job with specified savepoints. In SQL Client, it's allow to use `SET` command
+to specify the path to the savepoints.
 
-```yaml
-tables:
-  - name: MyTableSource
-    # ...
-  - name: MyRestrictedView
-    type: view
-    query: "SELECT MyField2 FROM MyTableSource"
-  - name: MyComplexView
-    type: view
-    query: >
-      SELECT MyField2 + 42, CAST(MyField1 AS VARCHAR)
-      FROM MyTableSource
-      WHERE MyField2 > 200
+```sql
+Flink SQL> SET execution.savepoint.path=/tmp/flink-savepoints/savepoint-cca7bc-bb1e257f0dab;
+[INFO] Session property has been set.
 ```
 
-Similar to table sources and sinks, views defined in a session environment file have highest precedence.
+When the path is specified, Flink will try to restore the state from the savepoint when execute statements.
 
-Views can also be created within a CLI session using the `CREATE VIEW` statement:
+SQL Client also supports to disable restore from the savepoint by `RESET` command.
```

Comment:
```suggestion
Because the savepoint path configuration will affect all the following DML statements, you can also use `RESET` command to reset this configuraiton, i.e. disable restoring from savepoint. 
```